src/AutoMerge/guidelines.jl

```diff
@@ -206,9 +235,27 @@ end
 damerau_levenshtein(name1, name2) = StringDistances.DamerauLevenshtein()(name1, name2)
 sqrt_normalized_vd(name1, name2) = VisualStringDistances.visual_distance(name1, name2; normalize=x -> 5 + sqrt(x))
 
-const guideline_distance_check =
-    Guideline("Name is not too similar to existing package names",
-              data -> meets_distance_check(data.pkg, data.registry_master))
+const guideline_distance_check = Guideline(;
+    info = "Name is not too similar to existing package names",
+    docs = """
+  To prevent confusion between similarly named packages, the names of new packages must also satisfy the following three checks: (for more information, please see the \"Name similarity distance check\" subsection under the \"Additional information\" section below)
```

Comment:
```suggestion
  To prevent confusion between similarly named packages, the names of new packages must also satisfy the following three checks: (for more information, please see the \"Name similarity distance check\" subsection under \"Additional information\" below)
```
I thought this would be slightly shorter while still being clear.